### DISCUSSION TITLE
Where do we report issues/false positives for parental control and browsing security?

### DISCUSSION THREAD

**Discussion by timkgh** [AdguardTeam/AdGuardHome#3624] [Q&A] [answered]
E.g. `f002.backblazeb2.com` (and possibly other subdomains) is blocked as an adult website but that's just a generic download URL for Backblaze B2 storage. Granted, there could be adult content in some bucket, but it cannot be blocked in general, it breaks valid backup tools.

**Comment by timkgh** [comment]
I reported it here: https://github.com/AdguardTeam/AdguardFilters/issues/95236

**Comment by ameshkov** [comment, marked as answer]
For parental control and browsing security the proper channel is `support@adguard.com`.

The filters repo also works okay, though.

**Comment by timkgh** [comment]
@ameshkov could you please take a look at this comment? thanks.
https://github.com/AdguardTeam/AdguardFilters/issues/95236#issuecomment-923320142
